pkg/gateway/gateway.go

```diff
@@ -23,11 +25,14 @@ func New(cfg Config) (*Gateway, error) {
 
 	sm := newServiceMapper(kc)
 
+	nginxCfg := newNGINXConfig(cfg.NGINXHealthPort)
 	var nm NGINXManager
 	if cfg.NGINXDryRun {
+		log.Printf("Creating loggingNGINXManager with NGINXConfig: %+v", nginxCfg)
```

Comment:
This log line is sort of confusing. We've constructed an NGINXConfig, but it's not actually being used by loggingNGINXManager. What would you think about a single log line below line 28 like this:

```
log.Printf("Using nginx config: %+v", nginxCfg)
```